__snapshots__/IsEmptyInterface

```diff
@@ -1,15 +1,14 @@
-sources/IsEmptyInterface.test.ts(11,27): error TS2554: Expected 1 arguments, but got 0.
 Files:            54
-Lines:         61530
-Nodes:        180467
-Identifiers:   62604
-Symbols:      104007
-Types:         36924
-Memory used: 135129K
-I/O read:      0.02s
+Lines:         61493
+Nodes:        180360
+Identifiers:   62571
+Symbols:      103235
+Types:         36774
+Memory used: 153131K
+I/O read:      0.01s
 I/O write:     0.00s
-Parse time:    0.76s
-Bind time:     0.22s
-Check time:    3.14s
+Parse time:    0.66s
+Bind time:     0.28s
+Check time:    4.42s
 Emit time:     0.00s
-Total time:    4.12s
+Total time:    5.37s
```

Comment:
Did it get slower?  That's odd.

Reply:
It's just a single run. When averaging 10 runs it did get slightly faster (6%). The snapshot is more for the other diagnostics (see __snapshots__/README.md)